web/cypress/e2e/plugin/create-delete-in-drawer-plugin.cy.js

```diff
@@ -292,12 +292,12 @@ context('Delete Plugin List with the Drawer', () => {
       cy.contains(item)
         .parents(selector.pluginCardBordered)
         .within(() => {
-          cy.get('button').click();
+          cy.get('button', 'Enable').click();
```

Comment:
Well, I thought this would be more stable, and I would cancel the 'enable'. 

> As for "this E2E test actually disabled the plugins and then attempted to check if the plugin was enabled (or switch is in" on "state) which is causing failure in CI." 

what do you think should be solved? You can change it directly in my PR
@butterfly1924 
cc @bzp2010